crates/storage/libmdbx-rs/src/txn_manager.rs

```diff
@@ -190,12 +216,14 @@ mod read_transactions {
         /// We store `usize` instead of a raw pointer as a key, because pointers are not
         /// comparable. The time of transaction opening is stored as a value.
         active: DashMap<usize, Instant>,
-        /// List of read transactions aborted by the [ReadTransactions::start_monitor].
-        /// We keep them until user tries to abort the transaction, so we're able to report a nice
+        /// List of read transactions aborted by the [ReadTransactions::start_monitor] or the user.
```

Comment:
hmm, unsure... If user aborts the transaction, it's fine and all works just as without the `read-tx-timeouts` feature. But if we abort the transaction due a timeout, we need to place it in the list of aborted transactions to report a nice error.

Reply:
ok got it, wasn't quite sure based on the issue what the desired behaviour was